tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,4769 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #83.        print(f"Workers:{n}, Sample Run:{i}, Finished loading data in {sample_record['data_read']}s")
</pre><p>The time should be seconds instead on ns to make ti more readable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>